jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -127,6 +128,20 @@ public PermissionConfiguration() {}
     }
   }
 
+  /** Configuration for {@code platform} parameter. */
```

Comment:
Add comments about default perhaps?

Reply:
What would an appropriate comment look like for this constructor?Would this work?

`/** Initializing a default platform in case a user doesn't pass in one **/

Reply:
I think for IDE users, just seeing the pop up of: 
```
/** Configuration for {@code platform} parameter. Defaults to amd64/linux. **/
```
 is helpful, since they wont have to go into the code and see what the defaults are?

But commenting sometimes feels pretty subjective. This is just how I would do it.

Reply:
I agree commenting is subjective sometimes.I think your suggested comment works well in this case.